packages/features/bookings/lib/handleCancelBooking.ts

```diff
@@ -626,7 +623,11 @@ async function handler(req: CustomRequest) {
     });
 
     // We skip the deletion of the event, because that would also delete the payment reference, which we should keep
-    await apiDeletes;
+    try {
+      await apiDeletes;
+    } catch (error) {
+      console.error("Error deleting event", error);
```

Comment:
WOuld it be better to use logger in all these instances of console.error?

Reply:
It depends on what are we trying to achieve with logger. If we use a report tooling, it should take care wether we use console.error or logger IMO.